### DISCUSSION TITLE
Workflow waiting for approval from pull request.

### DISCUSSION THREAD

**Discussion by TrustBetOC** [community/community#77825] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

Hello all!

I have a question I hope someone can help with. I created a pull request about a week ago and while the preview and all checks are good, I still have the message "1 workflow awaiting approval. This workflow requires approval from a maintainer."
I have reviewed my work, and am at a bit of loss as to what to do. The workflow is just a basic personalization of my DAO on Realm.today, not a big one, just my first.
Any help would be very much appreciated.

**Comment by SonnyTaylor** [comment]
It seems like the workflow in your pull request requires approval from a maintainer before it can be merged. Check whos a maintainer in it and try get them to approve it.

**Comment by dakshithadissanayaka** [comment]
Sure, I can help you with that.

The message "1 workflow awaiting approval. This workflow requires approval from a maintainer" means that your pull request cannot be merged until a maintainer of the repository reviews and approves it. This is a common practice among many open-source projects to ensure that code changes are reviewed and approved by experienced developers before they are merged into the main codebase.

Here are some steps you can take to get your pull request approved:

1. **Review your pull request:** Carefully review your code changes and make sure they are well-written, documented, and follow the project's coding style guidelines. Double-check that your changes do not introduce any new bugs or regressions.

2. **Address any feedback:** If you have received any feedback or comments from reviewers, address them promptly and thoroughly. Provide explanations for your design decisions and address any concerns raised.

3. **Ping the maintainers:** If it has been a while since your pull request was opened, politely ping the maintainers to remind them about your pull request and ask for their review. You can do this by leaving a comment on the pull request or sending them a private message.

4. **Be patient:** Maintainers often have busy schedules and may not be able to review pull requests immediately. Be patient and allow them the time they need to thoroughly review your changes.

Here are some additional tips for getting your pull request approved:

* **Make sure your pull request is small and focused:** Large pull requests can be more difficult for maintainers to review, so break your changes down into smaller, more manageable pull requests.

* **Write clear and concise commit messages:** Commit messages should describe the changes made in each commit and provide context for the reviewer.

* **Use a descriptive title for your pull request:** The title should clearly convey the purpose of your changes and make it easy for maintainers to understand what you are proposing.

* **Follow the project's contributing guidelines:** Each project may have specific guidelines for contributing code. Make sure you are familiar with and follow these guidelines to ensure a smooth review process.

I hope this helps!

**Comment by TrustBetOC** [comment]
Thanks guys, I'll have a look at it.
